styles/_editorjs.scss

```diff
@@ -107,6 +107,15 @@
     }
   }
 
+  .ce-inline-tool-input {
+    background-color: var(--darkGray);
```

Comment:
<img width="386" alt="Screenshot 2023-04-20 at 21 37 12" src="https://user-images.githubusercontent.com/55176773/233431740-fd657237-9b68-4463-bfdc-e05f715971a5.png">
1. link input background color should NOT be so, select another
2. borders are not shown - BUG